mobile/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -527,15 +543,15 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 86;
-				MARKETING_VERSION = "$(FLUTTER_BUILD_NAME)";
-				DEVELOPMENT_TEAM = 2F67MQ8R79;
+				DEVELOPMENT_TEAM = 6WU78265ZG;
```

Comment:
@alextran1502 I had to change this to compile it here. You should change it back by checking it out yourself and going to Runner -> Targets -> Signing & Capabilities -> Signing, then changing it back correctly. I'm hesitant of mangling the `project.pbxproj` files by editing it myself. I think all that needs done is to change back the DEVELOPMENT_TEAM.

Alternatively, maybe you should add my Apple Developer Account to your Development Team, so I can compile it without changing this again.